hack/ci-kind-molecule-tests.sh

```diff
@@ -0,0 +1,441 @@
+    failures during any one of the tests.
+    Default: false
+HELP
+}
+
+# process command line arguments
+while [[ $# -gt 0 ]]; do
+  key="$1"
+  case $key in
+    -at|--all-tests)              ALL_TESTS="$2";             shift;shift; ;;
+    -ce|--client-exe)             CLIENT_EXE="$2";            shift;shift; ;;
+    -dorp|--docker-or-podman)     DORP="$2";                  shift;shift; ;;
+    -gcp|--git-clone-protocol)    GIT_CLONE_PROTOCOL="$2";    shift;shift; ;;
+    -hb|--helm-branch)            HELM_BRANCH="$2";           shift;shift; ;;
+    -h|--help)                    helpmsg;                    exit 1       ;;
+    -hf|--helm-fork)              HELM_FORK="$2";             shift;shift; ;;
+    -ii|--install-istio)          INSTALL_ISTIO="$2";         shift;shift; ;;
+    -ir|--irc-room)               IRC_ROOM="$2";              shift;shift; ;;
+    -iv|--istio-version)          ISTIO_VERSION="$2";         shift;shift; ;;
+    -kb|--kiali-branch)           KIALI_BRANCH="$2";          shift;shift; ;;
+    -kf|--kiali-fork)             KIALI_FORK="$2";            shift;shift; ;;
+    -kob|--kiali-operator-branch) KIALI_OPERATOR_BRANCH="$2"; shift;shift; ;;
+    -kof|--kiali-operator-fork)   KIALI_OPERATOR_FORK="$2";   shift;shift; ;;
+    -lb|--logs-branch)            LOGS_BRANCH="$2";           shift;shift; ;;
+    -lf|--logs-fork)              LOGS_FORK="$2";             shift;shift; ;;
+    -lpn|--logs-project-name)     LOGS_PROJECT_NAME="$2";     shift;shift; ;;
+    -oapi|--openshift-api)        OPENSHIFT_API="$2";         shift;shift; ;;
+    -oi|--operator-installer)     OPERATOR_INSTALLER="$2";    shift;shift; ;;
+    -sd|--src-dir)                SRC="$2";                   shift;shift; ;;
+    -st|--skip-tests)             SKIP_TESTS="$2";            shift;shift; ;;
+    -sv|--spec-version)           SPEC_VERSION="$2";          shift;shift; ;;
+    -udi|--use-dev-images)        USE_DEV_IMAGES="$2";        shift;shift; ;;
+    -ul|--upload-logs)            UPLOAD_LOGS="$2";           shift;shift; ;;
+    *) echo "Unknown argument: [$key]. Aborting."; helpmsg; exit 1 ;;
+  esac
+done
+
+# abort on any error
+set -e
+
+# set up some of our defaults
+CLIENT_EXE=${OC:-/usr/bin/kubectl}
+SRC="${SRC:-/tmp/KIALI-GIT}"
+DORP="${DORP:-docker}"
+GIT_CLONE_PROTOCOL="${GIT_CLONE_PROTOCOL:-git}"
+
+KIND_NAME="${KIND_NAME:-ci}"
+
+# if you want to test code from different forks and/or branches, set them here
+HELM_FORK="${HELM_FORK:-kiali/helm-charts}"
+HELM_BRANCH="${HELM_BRANCH:-master}"
+KIALI_FORK="${KIALI_FORK:-kiali/kiali}"
+KIALI_BRANCH="${KIALI_BRANCH:-master}"
+KIALI_OPERATOR_FORK="${KIALI_OPERATOR_FORK:-kiali/kiali-operator}"
+KIALI_OPERATOR_BRANCH="${KIALI_OPERATOR_BRANCH:-master}"
+
+# details about the github repo where the logs are to be stored
+LOGS_PROJECT_NAME="${LOGS_PROJECT_NAME:-kiali-molecule-test-logs}"
+LOGS_FORK="${LOGS_FORK:-jmazzitelli}"
```

Comment:
if we run this from github CI, we'll want to change this to somewhere else where we can store the logs - probably under the kiali umbrella

Reply:
I would prefer printing logs to the terminal, so that you can go to the _Checks_ tab of GitHub and inspect the logs there like any regular CI, rather than going to another place to look for the logs.